zone/doors.cpp

```diff
@@ -325,19 +328,30 @@ void Doors::HandleClick(Client* sender, uint8 trigger) {
 		 */
 		else if (lock_pick_item != nullptr) {
```

Comment:
Can also change this to if lock_pick_item versus explicit nullptr check.

Reply:
See above - changed as little as I needed to in order to keep it simple.  Let me know what the consensus is.